Library/Formula/owfs.rb

```diff
@@ -2,28 +2,16 @@
 
 class Owfs < Formula
   homepage 'http://owfs.org/'
-  url 'https://downloads.sourceforge.net/project/owfs/owfs/2.9p1/owfs-2.9p1.tar.gz'
-  version '2.9p1'
-  sha1 'd431ae5b8eec6988f8ee1e401325eba845cf3696'
+  url 'http://sourceforge.net/projects/owfs/files/owfs/3.1p0/owfs-3.1p0.tar.gz'
+  version '3.1p0'
+  sha256 '62fca1b3e908cd4515c9eb499bf2b05020bbbea4a5b73611ddc6f205adec7a54'
 
   depends_on 'libusb-compat'
 
   def install
-    # Fix include of getline and strsep to avoid crash
-    inreplace 'configure', '-D_POSIX_C_SOURCE=200112L', ''
-
-    # 'tac' command is missing in MacOSX
-    inreplace 'src/man/Makefile.am', 'tac', 'tail -r'
-    inreplace 'src/man/Makefile.in', 'tac', 'tail -r'
-
     system "./configure", "--disable-debug",
                           "--disable-dependency-tracking",
-                          "--disable-swig",
-                          "--disable-owfs",
                           "--disable-owtcl",
-                          "--disable-zero",
```

Comment:
Our test-bots are clean machines, and we don't permit things to just scoop up undeclared dependencies at their free-will to prevent a myriad of problems. If removing the flags means it checks for dependencies, we need to declare those.